tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
     py{37,38,39,310}-dj-3.2
     py{38,39,310}-dj-4.0
     py{310}-dj-master
-    flake8
+    ruff
```

Comment:
This is I think not at all in use by tox, or am I wrong? I think we would need an additional job and also additional config in GH actions, invoking `pre-commit` see example here https://github.com/ansible/ansible-lint/blob/main/tox.ini#L74 (probably a separated PR)

Reply:
good point, thanks for the pointers.

Reply:
Actually, we run pre-commit.ci and it is a blocking check on GitHub, so no need to add anything here, it was just requiring to be removed here :-)